conf/messages.cy

```diff
@@ -1,6 +1,6 @@
 service.name = customs-service-status-frontend
 service.text = Dyma'ch gwasanaeth newydd
 
-service.state.AVAILABLE=AVAILABLE
-service.state.UNAVAILABLE=UNAVAILABLE
-service.state.UNKNOWN=UNKNOWN
+service.state.AVAILABLE=Available
+service.state.UNAVAILABLE=Unavailable
```

Comment:
should be `Known issues`

Reply:
this is being replaced by the Welsh, but have corrected.